src/renderer/services/eth2/client/eth2ApiClient/cgEth2BeaconPoolApi.ts

```diff
@@ -24,6 +24,7 @@ export class CgEth2BeaconPoolApi implements IBeaconPoolApi {
                     toHex(attestation.data.beaconBlockRoot),
                     attestation.data.index,
                     attestation.data.slot,
+                    attestation.aggregationBits.findIndex((bit) => Number(bit) === -1),
```

Comment:
```suggestion
                    attestation.aggregationBits.findIndex((bit) => bit === true),
```

Reply:
we talk about this... for some reason we got `-true` what is `-1`, if i try to compare `bit === -true` got type error i cannot compare boolean with number

Reply:
I don't think that's possible could you log aggregation bits and post logs here?